### DISCUSSION TITLE
[Feature] immich webui should function as a homescreen web app on ios

### DISCUSSION THREAD

**Discussion by lowlyw** [immich-app/immich#1987] [Feature Request]
### The feature

on iPad, i want to view my immich webui, but not install the app (all the syncing happens through my phone). I would like to be able to add immich as a homescreen web app ( https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html ) to behave better on iPad. 

It is minimal changes, just to app.html i believe https://github.com/immich-app/immich/blob/main/web/src/app.html

Adding this feature request as more of a discussion on if you believe it should be done, as the actual changes are very minimal. I can't think of any reason not to, but perhaps you can.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
Hello, feel free to open a PR for this feature

**Comment by bo0tzz** [comment]
#2310
